tests/fuzz/cbor_double_encode_decode.c

```diff
@@ -0,0 +1,66 @@
+/**
```

Comment:
does llvm fuzzer generate completely random data here as input? or it has some idea of what values are valid in cbor?

Reply:
I believe it is completely random, https://llvm.org/docs/LibFuzzer.html#id23

But, for sure, it doesn't know anything about cbor

Reply:
so it will ignore anything malformed? how often will it generate malformed garbage vs something thats parsable?

just want to verify we are actually getting useful results out of it